codex/blocktype.nim

```diff
@@ -37,91 +39,44 @@ type
     cid*: Cid
```

Comment:
I think this can simplify things a lot, if we make the `cid` and `BlockAddress`?

Reply:
Not really, block can belong to more than a single tree.